app/views/layouts/_head.html.erb

```diff
@@ -36,6 +36,17 @@
   <%= include_account_css %>
   <script>
     window.bz_custom_css_url = <%= custom_css_url.to_json.html_safe %>;
+    try {
+      if(window.location.pathname != '/' && window.location.pathname.indexOf('/login') != 0) {
+        var message = "";
+        message += 'last_url=' + encodeURIComponent(window.location.href) + "&";
+        message += 'last_url_title=' + encodeURIComponent(document.title);
+        var request = new XMLHttpRequest();
+        request.open("POST", "/bz/last_user_url", true);
```

Comment:
Yikes.  This worries me a little from a performance perspective.  Is there no MVC way to accomplish this instead of using javascript to post to the route?